### PR TITLE
Add two command options to add user/client

### PR DESCRIPTION
**Pull request by IsaacYangSLA**

### CODE REVIEW

File: nvflare/lighter/provision.py [hunk 1]
```diff
@@ -25,6 +25,30 @@
...
+        args:
+        gpu_resource_key: gpu
+"""
+
+adding_user_error_msg = """
+name: $USER_EMAIL_ADDRESS
+org: $ORGANIZATION_NAME
+roles:
+    - $ROLE
+"""
```
Comment (YuanTingHsieh) [inline nvflare/lighter/provision.py:50]: I am thinking if we put these "example yaml" in separate files.
Then in the future if we are going to modify the content we don't need to change the code in this file.

File: nvflare/lighter/provision.py [hunk 2]
```diff
@@ -86,6 +112,27 @@ def main():
...
+            print("The yaml file format is")
+            print(adding_user_error_msg)
+            exit(0)
+    if args.add_client:
+        try:
+            extra = load_yaml(os.path.join(current_path, args.add_client))
+            extra.update({"type": "client"})
+            participants.append(Participant(**extra))
+        except BaseException as e:
+            print("Error during adding client")
```
Comment (YuanTingHsieh) [inline nvflare/lighter/provision.py:131]: this message is not consistent with "** Error during adding user **"

suggest unify their style.

File: nvflare/lighter/provision.py (hunk 1 above)
Comment (IsaacYangSLA) [reply nvflare/lighter/provision.py:50]: My thought was that the info for clients/users were still undecided yet.  After everything is finalized, this user-facing msg can be in a separate file.

File: nvflare/lighter/provision.py (hunk 2 above)
Comment (IsaacYangSLA) [reply nvflare/lighter/provision.py]: Thanks.  Fixed.

**Review by YuanTingHsieh — APPROVED** [review]
LGTM, for the message we can discuss and finalize it later.

**Comment by YuanTingHsieh** [comment]
@IsaacYangSLA the check says it "requires all commits to be signed".
Can you sign the commits, thanks

**Review by nvkevlu — APPROVED** [review]
Thanks, looks pretty good, maybe the message could be:

The yaml file format should be

instead of

The yaml file format is

**Pull request merged**